### PR TITLE
Feature/dev 9 setup module structure

### PR DESCRIPTION
**Pull request by Stualyttle**

### CODE REVIEW

File: Client/src/app/events/interactionCreate.ts
Comment (pgm-jeflerou) [inline Client/src/app/events/interactionCreate.ts:1]: whole file is in comments ?

File: Client/src/deployCommands.ts
Comment (pgm-jeflerou) [inline Client/src/deployCommands.ts:1]: Whole file in comments?

File: Client/src/app/events/interactionCreate.ts
Comment (Stualyttle) [reply Client/src/app/events/interactionCreate.ts:1]: (Deploy commands also)

File: Client/src/deployCommands.ts
Comment (Stualyttle) [reply Client/src/deployCommands.ts:1]: Added comment in code

**Review by pgm-jeflerou — APPROVED** [review]

**Pull request merged**